src/linagora.esn.unifiedinbox/app/providers.js

```diff
@@ -327,4 +327,12 @@ angular.module('linagora.esn.unifiedinbox')
 
       return item;
     };
+  })
+
+  .factory('getDynamicTranslatedValue', function() {
```

Comment:
Add doc for this please

`getDynamicTranslatedValue` is not a correct name I think